src/containers/DeveloperPortal/containers/SdkAndLibraries/components/DeveloperPortalLayout/DeveloperPortalLayout.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../shared';
+@import 'styles/z-index';
+
+.DeveloperPortalLayout {
```

Comment:
same for this, will it be conflicting with the existing `DeveloperPortalLayout`?